aries-site/src/examples/components/menu/MenuBatchActionsExample.js

```diff
@@ -1,63 +1,74 @@
 import PropTypes from 'prop-types';
-import { useContext, useEffect } from 'react';
-import { Box, DataTable, Menu, ResponsiveContext, Text } from 'grommet';
+import { useContext, useState, useEffect } from 'react';
+import {
+  Box,
+  Data,
+  DataFilters,
+  DataTable,
+  DataSearch,
+  DataSummary,
+  Menu,
+  ResponsiveContext,
+  Text,
+  Toolbar,
+} from 'grommet';
 import {
   StatusWarningSmall,
   StatusCriticalSmall,
   StatusGoodSmall,
   StatusUnknownSmall,
 } from 'grommet-icons';
 
-import {
-  FilterControls,
-  FiltersProvider,
-  useFilters,
-} from '../../templates/FilterControls';
-
 const { servers } = require('../../../data/mockData/servers.json');
 
-const filtersConfig = [];
-
 export const MenuBatchActionsExample = ({ containerRef }) => {
   // containerRef is for demonstration purposes on this site. Most
   // implementations should likely remove.
 
   const layerProps = {
     // containerRef is for demonstration purposes on this site. Most
     // implementations should likely remove.
-    target: containerRef && containerRef.current,
```

Comment:
@taysea here is where I have the Layer props lmk your thoughts

Reply:
I agree with removing `layerProps` altogether and just doing `<DataFilters layer />` below.

Reply:
same sounds good